packages/microservices/server/server-kafka.ts

```diff
@@ -245,7 +245,7 @@ export class ServerKafka extends Server<never, KafkaStatus> {
         'Not initialized. Please call the "listen"/"startAllMicroservices" method before accessing the server.',
       );
     }
-    return this.client as T;
+    return [this.client, this.consumer, this.producer] as T;
```

Comment:
Instead of modifying the "unwrap" method, we should add getters for consumer and producer, similar to ClientKafka class

Reply:
sounds much better. But what `unwrap` method should return then? This?

```suggestion
    return this;
```

I guess you meant getters/setters must be defined on `ServerKafka`, right?

Reply:
> sounds much better. But what unwrap method should return then? This instance?

this.client is OK 

> I guess you meant getters/setters must be defined on ServerKafka, right?

just getters. I meant that they should be defined in ServerKafka, similar to what we already have in ClientKafka